src/bundle.c

```diff
@@ -411,10 +411,13 @@ int show_bundle_reqd_by(const char *bundle_name, bool server)
  *  	performing a unlink(2) for each filename.
  *  6) Done.
  */
-int remove_bundle(const char *bundle_name)
+int remove_bundles(char **bundles)
```

Comment:
Swupd in general isn't strict about having const in variables that aren't changed in the function, but as this function was already adding a const, I would keep that.

Reply:
Also update the header documentation saying that the function now handle multiple bundles

Reply:
Good point.

Reply:
Turns out the compiler didn't like that, after adding the const I got a compiler warning:

src/clr_bundle_rm.c: In function ‘bundle_remove_main’:
src/clr_bundle_rm.c:200:24: warning: passing argument 1 of ‘remove_bundles’ from incompatible pointer type [-Wincompatible-pointer-types]
  return remove_bundles(bundles);
                        ^~~~~~~
In file included from src/clr_bundle_rm.c:35:
src/swupd.h:380:40: note: expected ‘const char **’ but argument is of type ‘char **’
 extern int remove_bundles(const char **bundles);
                           ~~~~~~~~~~~~~^~~~~~~

Reply:
The compiler didn't like it because you removed `const` from `src/swupd.h`

Reply:
/me reads you error message closer.. nvm